bc_obps/reporting/models/rls_configs/report.py

```diff
@@ -0,0 +1,19 @@
+from reporting.enums.enums import ReportingTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [
+            RlsOperations.SELECT,
+            RlsOperations.INSERT,
+            RlsOperations.UPDATE,
+            RlsOperations.DELETE,
```

Comment:
Do we need "DELETE" here? would reports be archived if we ever needed to re-create one?

Reply:
Won't they need delete for actions that fully delete a report? Like changing registration purpose?

Reply:
I guess you're right! Although we wouldn't be tossing away entire reports if they changed registration purpose, maybe just versions? Anyways, it's their data, so we can give them that option